include/vk_mem_alloc.h

```diff
@@ -1035,6 +1052,11 @@ typedef struct VmaVulkanFunctions
     /// Fetch from "vkGetDeviceImageMemoryRequirements" on Vulkan >= 1.3, but you can also fetch it from "vkGetDeviceImageMemoryRequirementsKHR" if you enabled extension VK_KHR_maintenance4.
     PFN_vkGetDeviceImageMemoryRequirementsKHR VMA_NULLABLE vkGetDeviceImageMemoryRequirements;
 #endif
+#ifdef VK_USE_PLATFORM_WIN32_KHR
+    PFN_vkGetMemoryWin32HandleKHR VMA_NULLABLE vkGetMemoryWin32HandleKHR;
+#else
+    void* VMA_NULLABLE vkGetMemoryWin32HandleKHR;
```

Comment:
For other extensions we don't define its functions when the extension is not available. Why do you propose this? I think it would be better if any usage of this function is also surrounded by `#ifdef VK_USE_PLATFORM_WIN32_KHR` so this is not needed.

Reply:
ABI compatibility, for other extensions it is normal to be defined on the scale of program. This one however is not always defined. I do the same in my library. I have dedicated lib for platform extensions, which include <vulkan/vulkan_win32.h>
and it will behave normal without the extensions.